components/AnswerInput.tsx

```diff
@@ -62,16 +62,36 @@ export default function AnswerInput({ categoryId }: Props) {
       onSubmit();
     }
   };
-
-  return (
-    <Container>
-      <input
-        placeholder="מה הקשר?"
-        value={value}
-        onKeyPress={onKeyPress}
-        onChange={(e) => setValue(e.target.value)}
-      />
-      <button onClick={onSubmit}>בום</button>
-    </Container>
-  );
+  if (board.clues[categoryId] === undefined || board.clues[categoryId] === "") {
+    return (
+      <Container>
+        <input
+          placeholder="מה הקשר?"
+          value={value}
+          onKeyPress={onKeyPress}
+          onChange={(e) => setValue(e.target.value)}
+        />
+        <button onClick={onSubmit}>בום</button>
+      </Container>
+    );
+  } else {
+    return (
+      <Container>
+        <input
+          placeholder="מה הקשר?"
+          value={value}
+          onKeyPress={onKeyPress}
+          onChange={(e) => setValue(e.target.value)}
+        />
+        <button onClick={onSubmit}>בום</button>
+        <button
+          onClick={() => {
+            alert(board.clues[categoryId]);
+          }}
+        >
+          💡 רמז
+        </button>
```

Comment:
Please make sure they're the same height. 

![image](https://user-images.githubusercontent.com/2545447/116466409-19103800-a877-11eb-9397-2bc46f41f7b6.png)